Cannot uninstall requirement tinymce, not installed
Exception information:
Traceback (most recent call last):
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/basecommand.py", line 115, in main
    self.run(options, args)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/commands/uninstall.py", line 37, in run
    requirement_set.uninstall(auto_confirm=options.yes)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 763, in uninstall
    req.uninstall(auto_confirm=auto_confirm)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 378, in uninstall
    raise UninstallationError("Cannot uninstall requirement %s, not installed" % (self.name,))
UninstallationError: Cannot uninstall requirement tinymce, not installed
------------------------------------------------------------
/Users/ebone/.virtualenvs/easymode/bin/pip run on Sun May 30 20:52:43 2010
Unpacking /Users/ebone/Sites/django-easymode/requirements.txt
  Cannot unpack file /Users/ebone/Sites/django-easymode/requirements.txt (downloaded from /var/folders/6O/6O7oziQ3F-eKVGF4-kQQl++++TM/-Tmp-/pip-6Xsfrn-build, content-type: text/plain); cannot detect archive format
Cannot determine archive format of /var/folders/6O/6O7oziQ3F-eKVGF4-kQQl++++TM/-Tmp-/pip-6Xsfrn-build
Exception information:
Traceback (most recent call last):
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/basecommand.py", line 115, in main
    self.run(options, args)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/commands/install.py", line 155, in run
    requirement_set.install_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 830, in install_files
    self.unpack_url(url, location, self.is_download)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 918, in unpack_url
    self.unpack_file(source, location, content_type, link)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 1068, in unpack_file
    raise InstallationError('Cannot determine archive format of %s' % location)
InstallationError: Cannot determine archive format of /var/folders/6O/6O7oziQ3F-eKVGF4-kQQl++++TM/-Tmp-/pip-6Xsfrn-build
------------------------------------------------------------
/Users/ebone/.virtualenvs/easymode/bin/pip run on Tue Jun 15 23:17:44 2010
Exception:
Traceback (most recent call last):
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/basecommand.py", line 115, in main
    self.run(options, args)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/commands/install.py", line 153, in run
    for req in parse_requirements(filename, finder=finder, options=options):
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 1274, in parse_requirements
    filename, content = get_file_content(filename, comes_from=comes_from)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/util.py", line 300, in get_file_content
    f = open(url)
IOError: [Errno 2] No such file or directory: 'requirements.txt'
------------------------------------------------------------
/Users/ebone/.virtualenvs/easymode/bin/pip run on Tue Jun 15 23:17:47 2010
Exception:
Traceback (most recent call last):
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/basecommand.py", line 115, in main
    self.run(options, args)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/commands/install.py", line 153, in run
    for req in parse_requirements(filename, finder=finder, options=options):
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 1274, in parse_requirements
    filename, content = get_file_content(filename, comes_from=comes_from)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/util.py", line 300, in get_file_content
    f = open(url)
IOError: [Errno 2] No such file or directory: 'requirements.txt'
------------------------------------------------------------
/Users/ebone/.virtualenvs/easymode/bin/pip run on Tue Aug 10 12:17:48 2010
Cannot uninstall requirement easymode, not installed
Exception information:
Traceback (most recent call last):
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/basecommand.py", line 115, in main
    self.run(options, args)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/commands/uninstall.py", line 37, in run
    requirement_set.uninstall(auto_confirm=options.yes)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 763, in uninstall
    req.uninstall(auto_confirm=auto_confirm)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 378, in uninstall
    raise UninstallationError("Cannot uninstall requirement %s, not installed" % (self.name,))
UninstallationError: Cannot uninstall requirement easymode, not installed
------------------------------------------------------------
/Users/ebone/.virtualenvs/easymode/bin/pip run on Tue Aug 10 15:27:44 2010
Unpacking /Users/ebone/Sites/django-easymode/hudson-requirements.txt
  Cannot unpack file /Users/ebone/Sites/django-easymode/hudson-requirements.txt (downloaded from /var/folders/6O/6O7oziQ3F-eKVGF4-kQQl++++TM/-Tmp-/pip-1w92rK-build, content-type: text/plain); cannot detect archive format
Cannot determine archive format of /var/folders/6O/6O7oziQ3F-eKVGF4-kQQl++++TM/-Tmp-/pip-1w92rK-build
Exception information:
Traceback (most recent call last):
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/basecommand.py", line 115, in main
    self.run(options, args)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/commands/install.py", line 155, in run
    requirement_set.install_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 830, in install_files
    self.unpack_url(url, location, self.is_download)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 918, in unpack_url
    self.unpack_file(source, location, content_type, link)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 1068, in unpack_file
    raise InstallationError('Cannot determine archive format of %s' % location)
InstallationError: Cannot determine archive format of /var/folders/6O/6O7oziQ3F-eKVGF4-kQQl++++TM/-Tmp-/pip-1w92rK-build
------------------------------------------------------------
/Users/ebone/.virtualenvs/easymode/bin/pip run on Mon Aug 30 22:50:04 2010
Cannot uninstall requirement reversion, not installed
Exception information:
Traceback (most recent call last):
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/basecommand.py", line 115, in main
    self.run(options, args)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/commands/uninstall.py", line 37, in run
    requirement_set.uninstall(auto_confirm=options.yes)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 763, in uninstall
    req.uninstall(auto_confirm=auto_confirm)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 378, in uninstall
    raise UninstallationError("Cannot uninstall requirement %s, not installed" % (self.name,))
UninstallationError: Cannot uninstall requirement reversion, not installed
------------------------------------------------------------
/Users/ebone/.virtualenvs/easymode/bin/pip run on Sun Oct 10 10:37:16 2010
Cannot uninstall requirement django-reversion, not installed
Exception information:
Traceback (most recent call last):
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/basecommand.py", line 115, in main
    self.run(options, args)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/commands/uninstall.py", line 37, in run
    requirement_set.uninstall(auto_confirm=options.yes)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 763, in uninstall
    req.uninstall(auto_confirm=auto_confirm)
  File "/Users/ebone/.virtualenvs/easymode/lib/python2.6/site-packages/pip-0.6.3-py2.6.egg/pip/req.py", line 378, in uninstall
    raise UninstallationError("Cannot uninstall requirement %s, not installed" % (self.name,))
UninstallationError: Cannot uninstall requirement django-reversion, not installed
